src/tests/stdlib/memcmp_test.c

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file
+ * @brief Test unit for string/memcmp
+ *
+ * @date March 18,2020
+ * @author Sha Ahammed Roze
+ */
+
+ #include <embox/test.h>
+ #include <strings.h>
+
+ EMBOX_TEST_SUITE("Test suite for memcmp()");
+
+TEST_CASE("Test that buf1 > buf2") {
```

Comment:
Test description has not enough information
"Test that  SWgaOtP43df2 > SWGaOTP12df0"